testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/cacheddc/CachedDcDomainTestCase.java

```diff
@@ -57,6 +57,7 @@
 
     private static final long TIMEOUT_S = TimeoutUtil.adjust(30);
     private static final long NOT_STARTED_TIMEOUT_S = TimeoutUtil.adjust(7);
+    private static final int TIMEOUT_SLEEP_MILLIS = 300;
```

Comment:
There I was thinking cutting it down by 200ms was being adventerous ;) Sure I'll make it 50.